theme/snippets/cart-item.liquid

```diff
@@ -1,12 +1,24 @@
+<!-- 
+  Products created by Shopify Bookings have variants for each session of the event or course. Each variant has the same title as the product title. 
```

Comment:
Sounds like something we should allow people to configure in Bookings, right? As in people should be able to name a Course's session

Reply:
Actually, scratch that, I'm wrong and I learned something I didn't know about Bookings! 

- It seems like Bookings creates one variant for a course or event, so if you purchase a course, you [purchase the first available and only variant](https://share.getcloudapp.com/Qwu5b4AK). 
- That variant's title is the title of the Course or Event in Bookings (e.g. "Bird Watching", NOT "Bird Watching - Session I")
- In the Cart, Shopify gives the item's title as "Product Title - Variant Title", so we're seeing titles for Bookings products like "Bird Watching - Bird Watching". **So in this PR, for Bookings products only, I'm showing the Product Title only (design request).**
- And yes you can name a course's session in Bookings!

Shopify:
![image](https://user-images.githubusercontent.com/43737723/130688204-1b3b6e7e-7d6e-4d10-a597-5ce1a4762609.png)

Bookings:
![image](https://user-images.githubusercontent.com/43737723/130688378-2e127931-2fdb-4969-a68e-ffbb4289b693.png)

Reply:
@sepowitz you know Bookings better than I do... am I understanding it correctly? :)